docs/maven-tutorial.md

```diff
@@ -0,0 +1,150 @@
+```
+
+Confirm that Nexus is running by navigating your browser to the new Cluster-IP
+at port 8081. The server may take a bit of time to start, be patient!
+
+> NOTE: Nexus comes pre-configured for the Central Repository, but you may need
+> others for your application. At this point, add any special repositories to
+> Nexus. Documentation on how to add resositories can be found
+> [here](https://books.sonatype.com/nexus-book/reference/config-sect-new-repo.html).
+
+#### Using Probes to Check for Success
+
+This is a good time to set up readiness and livenes probes. These will
+periodically check to see that Nexus is running properly. More information on
+probes can be found
+[here](https://docs.openshift.org/latest/dev_guide/application_health.html).
+
+```bash
+oc set probe dc/nexus \
+	--liveness \
+	--failure-threshold 3 \
+	--initial-delay-second 30 \
+	-- echo ok
+oc set probe dc/nexus \
+	--readiness \
+	--failure-threshold 3 \
+	--initial-delay-seconds 30 \
+	--get-url=http://nexus:8081/content/groups/public
+```
+
+#### Adding Persistence to Nexus
+
+> NOTE: If you don't want persistent storage, continue to
+> [Connecting to Nexus](#connecting-to-nexus). However, your cached
+> dependencies will be lost if the pod is restarted for any reason.
+
+Now we'll create a Persistent Volume Claim for Nexus, so that our dependencies
+will not be lost when the Pod running the server terminates. If you do not have
+admin access on your cluster, ask your system administrator to create a
+Read/Write Persistent Volume for you.  Otherwise, read
+[Managing Volumes in OpenShift](https://docs.openshift.org/latest/dev_guide/volumes.html)
+for instructions on creating a persistent volume.
+
+Add a Persistent Volume Claim to the Nexus Deployment Config.
+```bash
+oc volumes dc/nexus --remove --confirm
+oc volumes dc/nexus --add \
+	--name 'nexus-pvc' \
+	--type 'pvc' \
+	--mount-path '/sonatype-work/' \
+	--claim-name 'nexus-pv' \
+	--claim-size '1G'
+```
+
+This removes the previous ephemeral volume for the Deployment Config, and adds a claim
+for 1 gigabyte of persistent storage mounted at `/sonatype-work`, which is where our
+dependencies will be stored. Due to the change in configuration, the Nexus pod will
+be redeployed automatically.
+
+Verify that Nexus is running by refreshing the Nexus page in your browser. You
+can monitor the deployment's progress using
+```bash
+oc get pods -w
+```
+
+## Connecting to Nexus
+
+The next steps will be pointing the build process to use the new Nexus
+repository.  The rest of the tutorial will be using
+[this example repository](https://github.com/openshift/jee-ex.git)
+with `wildfly-100-centos7` as a builder, but these changes should work for any
+project.
+
+The example builder image, wildfly-100-centos7, has 'MAVEN_MIRROR_URL' as part
+of its environment, so we can use this to point our builder image to our Nexus
+repository. If your image does not have an environment variable, you may need to
```

Comment:
"If your image does not support consuming an environment variable to configure a Maven mirror,"